src/pages/send/confirm/confirm.ts

```diff
@@ -147,6 +147,9 @@ export class ConfirmPage extends WalletTabsChild {
   }
 
   ionViewWillEnter() {
+    if (this.navCtrl.getPrevious().name == 'SelectInvoicePage') {
+      this.navCtrl.remove(this.navCtrl.getPrevious().index);
+    }
```

Comment:
This makes it so the back button on confirm page does not go back to the `SelectInvoicePage`. This is important as `SelectInvoicePage` sets the PayProV2 `buyerTransactionCurrency` and once set the `buyerTransactionCurrency` cannot be changed.